javascript/ql/src/semmle/javascript/frameworks/AngularJS/ServiceDefinitions.qll

```diff
@@ -297,34 +297,29 @@ abstract private class CustomSpecialServiceDefinition extends CustomServiceDefin
 /**
  * Holds if `mce` defines a service of type `moduleMethodName` with name `serviceName` using the `factoryFunction` as the factory function.
  */
+bindingset[moduleMethodName]
 private predicate isCustomServiceDefinitionOnModule(
   DataFlow::CallNode mce, string moduleMethodName, string serviceName,
-  DataFlow::SourceNode factoryFunction
+  DataFlow::SourceNode factoryArgument
```

Comment:
Couldn't this be typed as `DataFlow::Node` to preserve the original behavior?

Reply:
I'll do another evaluation now that you've merged the SourceNode PR.